editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1997,10 +1991,6 @@ function fillSpyOnlyMetadata(
     return parentLayoutSystem == null
   })
 
-  // Sort and then reverse these, so that lower level elements (with longer paths) are handled ahead of their parents
-  // Sort and then reverse these, so that lower level elements (with longer paths) are handled ahead of their parents
-  // and ancestors. This means that if there are a grandparent and parent which both lack global frames
-  // then the parent is fixed ahead of the grandparent, which will be based on the parent.
```

Comment:
why remove these comments?

Reply:
remnant of a reverted refactor, readded the comments in https://github.com/concrete-utopia/utopia/pull/3483/commits/db7e55e812e57e8613e8adf2818540f02f099bdc